Source/App/CMakeLists.txt

```diff
@@ -4,56 +4,50 @@
 # 
 
 # APP Directory CMakeLists.txt
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/Bin/${CMAKE_BUILD_TYPE}/)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/Bin/${CMAKE_BUILD_TYPE}/)
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/Bin/${CMAKE_BUILD_TYPE}/)
 
 # Include Subdirectories
-include_directories (${PROJECT_SOURCE_DIR}/API/)
-include_directories (${PROJECT_SOURCE_DIR}/API/OpenMAX/IL/)
-include_directories (${PROJECT_SOURCE_DIR}/../Bin/${CMAKE_BUILD_TYPE}/)
-if(${COMPILE_AS_CPP} EQUAL 1) 
-
-	file(GLOB SourceFiles
-	    "*.c"
-	)
-
-	set_source_files_properties( 
-		${SourceFiles} 
-		PROPERTIES LANGUAGE CXX )
-endif(${COMPILE_AS_CPP} EQUAL 1)
-
+include_directories (${PROJECT_SOURCE_DIR}/Source/API/)
+include_directories (${PROJECT_SOURCE_DIR}/Bin/${CMAKE_BUILD_TYPE}/)
 
 # App Source Files
-add_executable (HevcEncoderApp 
+add_executable (SvtHevcEncApp
     EbAppConfig.c
     EbAppContext.c
     EbAppFifo.c
     EbAppMain.c
     EbAppProcessCmd.c
-    EbAppSei.c
     EbTime.c
+    ../API/EbApi.h
+    EbAppConfig.h
+    EbAppContext.h
+    ../API/EbErrorCodes.h
+    ../API/EbTime.h
```

Comment:
add_executable only need source files as input.

Reply:
for visual studio projects to have the .h files visible we need to add them to the list.